frontend/index.ts

```diff
@@ -2,11 +2,14 @@ import { Commands, Context, Router } from '@vaadin/router';
 
 import './components/main-layout/main-layout';
 import './components/list-view/list-view';
-import { isUserLoggedIn } from './generated/SecurityEndpoint';
 import { logout } from '@vaadin/flow-frontend';
 
 import './utils/lumo';
 
+const isUserLoggedIn = function() {
+  return !!localStorage.getItem('loggedIn');
```

Comment:
This will always return true except if the loggedIn entry has not been set (null) or if it's the empty string ''.
If using 'true' and 'false' string values then this check should be:
```suggestion
  return localStorage.getItem('loggedIn') === 'true';
```

or if using '1' and '0' like I suggest:
```suggestion
  return localStorage.getItem('loggedIn') === '1';
```

Reply:
thanks for the catch, prefer to use 'true', 'false', I think it's a bit easier to read